### PR TITLE
Created option to set a high-frequency cutoff in pycbc_make_skymap

### PR DESCRIPTION
**Pull request by MarlinSchaefer**
Added an optional argument to pycbc_make_skymap.
This argument is passed to pycbc_single_template and SingleCoincForGraceDB.

For SingleCoincForGraceDB to accept a high-frequency cutoff changes to pycbc/io/live.py were required. I think the place I put these changes at don't interfere with anything else. I also think that implementing the high-frequency cutoff the way done here will pass it to Baystar.

### CODE REVIEW

File: pycbc/io/live.py [hunk 1]
```diff
@@ -263,10 +265,16 @@ def __init__(self, ifos, coinc_results, **kwargs):
         for ifo in self.psds:
             psd = self.psds[ifo]
             kmin = int(kwargs['low_frequency_cutoff'] / psd.delta_f)
+            if kwargs['high_frequency_cutoff']:
```
Comment (ahnitz) [inline pycbc/io/live.py:268]: @MarlinSchaefer @titodalcanton I'm not sure we need to modify the psd we give to bayestar do we? Assumin git covers at least the range intended?

File: bin/pycbc_make_skymap [hunk 2]
```diff
@@ -143,6 +143,7 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, ifar, ifos, thresh_SNR,
         command.append("--spin1z"),command.append(str(spin1z))
         command.append("--spin2z"),command.append(str(spin2z))
         command.append("--low-frequency-cutoff"),command.append(str(f_low))
+        command.append("--high-frequency-cutoff"),command.append(str(f_upper))
```
Comment (ahnitz) [inline bin/pycbc_make_skymap:146]: Does this work if you don't provide the option? I.E. it will try to do str(None) and probably raise an error, won't it?

Comment (ahnitz) [reply bin/pycbc_make_skymap:146]: if this is the case, this should probably be in a if statement so that it is really optional.

Comment (titodalcanton) [reply bin/pycbc_make_skymap:146]: +1

File: bin/pycbc_make_skymap [hunk 3]
```diff
@@ -341,6 +343,8 @@ if __name__ == '__main__':
     parser.add_argument('--spin2z', type=float, required=True)
     parser.add_argument('--f-low', type=float,
                         help="lower frequency cut-off (float)", default=20.0)
+    parser.add_argument('--f-upper', type=float,
+                        help="upper frequency cut-off (float)", default=None)
```
Comment (titodalcanton) [inline bin/pycbc_make_skymap:347]: `default=None` can be removed, since it is... The default :)

File: pycbc/io/live.py (hunk 1 above)
Comment (titodalcanton) [reply pycbc/io/live.py:268]: I do not think this is required either.

Comment (titodalcanton) [reply pycbc/io/live.py:268]: @lpsinger does BAYESTAR care if the PSD extends past the `f_final` frequency?

**Comment by ahnitz** [comment]
@MarlinSchaefer It looks like you've addressed the specific comments we had.

**Pull request merged**